product-mini/platforms/darwin/main.c

```diff
@@ -280,7 +280,7 @@ int main(int argc, char *argv[])
                                dir_list, dir_list_size,
                                NULL, 0,
                                env_list, env_list_size,
-                               argv, argc);
+                               argv, argc, -1, -1, -1);
```

Comment:
If we add api wasm_runtime_set_wasi_args_ex, no need to change code here.

Reply:
wasm_runtime_init_wasi